migrate.py

```diff
@@ -318,6 +320,52 @@ def actually_deprecate(checkout_path):
 
             write_yaml_into_file_as_is(os.path.join(meta, 'deprecated.yml'), DEPRECATE[namespace][collection])
 
+
+def cleanup_targets(checkout_path):
+    dep_targets = {}
```

Comment:
Plz use
```suggestion
    dep_targets = defaultdict(set)
```
so that you'll not nave to do conditional inserts.